robot_controllers/src/cartesian_twist.cpp

```diff
@@ -255,13 +272,18 @@ void CartesianTwistController::update(const ros::Time& now, const ros::Duration&
   // Limit target position of joint
   for (unsigned ii = 0; ii < num_joints; ++ii)
   {
-    if (tgt_jnt_pos_(ii) > joints_[ii]->getPositionMax())
-    {
-      tgt_jnt_pos_(ii) = joints_[ii]->getPositionMax();
-    }
-    else if (tgt_jnt_pos_(ii) < joints_[ii]->getPositionMin())
+    if(ii==0||ii==1||ii==3||ii==5)
```

Comment:
rather than hardcoding the joints which are non-continuous, we should query for continuous. (you can see how we compute a "continuous" array at the start of the follow joint trajectory, and then it can be used to decide whether to apply limits -- https://github.com/fetchrobotics/robot_controllers/blob/8663b33f57807a0585602fbe824fec8ebaead253/robot_controllers/src/follow_joint_trajectory.cpp#L105)